node_modules/oae-core/members/js/members.js

```diff
@@ -46,7 +46,7 @@ define(['jquery', 'oae.core', 'jquery.history'], function($, oae) {
 
             var url = '/api/group/' + widgetData.context.id + '/members';
             if (query) {
-                url = '/api/search/members/'  + widgetData.context.id;
+                url = '/api/search/group-members-library/'  + widgetData.context.id;
```

Comment:
"members" search is going away in place of "members-library", which is effectively the same, but more permissive than what the old members search was. How difficult would it be for us to coordinate this change? If there absolutely needs to be a transition period, I suppose we could do that.